components/collapsible/collapsible.vue

```diff
@@ -51,10 +48,10 @@
     <dt-collapsible-lazy-show
       :id="id"
       ref="contentWrapper"
-      :aria-hidden="`${!isOpen}`"
+      data-qa="contentWrapper"
+      :aria-hidden="`${!isOpen && !$slots.contentOnCollapsed}`"
       :aria-labelledby="labelledBy"
-      :aria-label="ariaLabel"
-      :show="isOpen"
+      :is-expanded="isOpen"
```

Comment:
hmm doesn't appear to perform the transition from open to closed when using the "open" prop rather than just clicking directly. This doesn't seem to be a problem in Vue 2

Reply:
That's weird, I investigated this a bit and my best guess is that it has to do with our version of Storybook for Vue 3. I added an example in our `variants` component where the collapsible `open` prop updates based on button clicks and it works without issues.